pkg/configmap.yaml

```diff
@@ -18,7 +18,8 @@ data:
           failure_mode_allow: false
           enable_plaintext_mode: true
       authentication_plugin:
-        enabled: all
+        enabled:
+        - all
```

Comment:
1. also update the default ais image at the same time?
2. should we also update the release note?